### PR TITLE
 Add `Issue Assignment` to contrib guide

### PR DESCRIPTION
**Pull request by matriv**
- Explain our policy regarding issue assignments for community contributors.
- Fix lines in `Pull Requests` section to not exceed the max line length.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -62,14 +62,35 @@ incredibly valuable as well. Please `get in touch`_ with us to tell us what you
...
+Issue assignments
+=================
+
+In general we will not assign issues to community contributors. If you'd like to
+work on an issue you can just state it with a comment, so it's visible to
+everyone that you have started working on it, but this shouldn't block any other
+potential contributor to work on the same issue. If there are multiple pull
+requests opened, we will try to choose the one that is in a better shape. The
+reasoning behind this approach, is that we have experienced the case that people
+get assigned to an issue, without any progress for very long time periods, while
```
Comment (seut) [inline CONTRIBUTING.rst:74]:
```suggestion
get assigned to an issue without any progress for very long time periods, while
```

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -62,14 +62,35 @@ incredibly valuable as well. Please `get in touch`_ with us to tell us what you
...
+"blocking" other potential contributors from working on that issue.
+
+Note: Of course you're always welcome to ask questions on the issue that would
+help you progress with the implementation.
```
Comment (seut) [inline CONTRIBUTING.rst:78]:
```suggestion
help you to progress with the implementation.
```

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -62,14 +62,35 @@ incredibly valuable as well. Please `get in touch`_ with us to tell us what you
 like and don't like about CrateDB.
 
 
+Issue assignments
+=================
+
+In general we will not assign issues to community contributors. If you'd like to
```
Comment (seut) [inline CONTRIBUTING.rst:68]:
```suggestion
In general, we will not assign issues to community contributors. If you'd like to
```

**Review by seut — APPROVED** [review]
Thanks! just some grammar suggestions

**Comment by matriv** [comment]
> Thanks! just some grammar suggestions

Thank you! applied.

**Pull request merged**